homeassistant/components/bluetooth/passive_update_processor.py

```diff
@@ -177,17 +395,23 @@ class PassiveBluetoothDataProcessor(Generic[_T]):
     """
 
     coordinator: PassiveBluetoothProcessorCoordinator
-    data: PassiveBluetoothDataUpdate[_T]
+    data: PassiveBluetoothDataUpdate[_T] = PassiveBluetoothDataUpdate()
```

Comment:
Why initialize it? It's in class definition so will be the same across instances.

Reply:
Good catch, its leftovers from previous refactoring

fixed in c2751634aaeb729172944a6834ae74fb9aae5db5